packages/netlify-cms-widget-markdown/src/MarkdownControl/plugins/CopyPasteVisual.js

```diff
@@ -0,0 +1,44 @@
+import { Document } from 'slate';
+import { setEventTransfer } from 'slate-react';
+import base64 from 'slate-base64-serializer';
+import isHotkey from 'is-hotkey';
+import { slateToMarkdown, markdownToSlate, htmlToSlate, markdownToHtml } from '../../serializers';
+
+const CopyPasteVisual = ({ getAsset, resolveWidget }) => {
+  const handleCopy = (event, editor) => {
+    const markdown = slateToMarkdown(editor.value.fragment.toJS());
+    const html = markdownToHtml(markdown, { getAsset, resolveWidget });
+    setEventTransfer(event, 'text', markdown);
+    setEventTransfer(event, 'html', html);
+    setEventTransfer(event, 'fragment', base64.serializeNode(editor.value.fragment));
```

Comment:
Copy for text and html paste targets, as well as for Slate itself (copying and pasting within the editor).